src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb

```diff
@@ -204,6 +204,36 @@ class c
             End Using
         End Function
 
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestNullableDateTimeCompletion(showCompletionInArgumentLists As Boolean) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+<Document><![CDATA[
+using System;
+class c
+{
+    void goo(DateTime? d)
+    {
+        d?.ToString("hh:mm:$$");
+    }
```

Comment:
This test is an exact copy/paste of the above test except changing DateTime to be nullable.

Reply:
#47873 is opened to make sure the test fails without the fix.